math/pythagoras.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 @author beqakd
+ * @file
+ *
+ * A basic implementation of 3d distance between two point.
+ */
+
+#include <cassert>   // For testing(assert)
+#include <cmath>     // For std::pow(), std::sqrt() and so on
+#include <iostream>  // For io operations
+
+/**
+ * The distance between two points in a three dimensional - 3D - coordinate
+ * system. For this we have 3 point x, y, z.
+ * Also we created method int distance which takes points and returns distance.
+ */
+class point {
+ private:
+    int x;
+    int y;
+    int z;
+
+ public:
+    point(int a, int b, int c)
```

Comment:
The constructor function to be documented

Reply:
You can make this generic by making it 2d as well by setting the third variable default value =0

Reply:
Its good idea.